### PR TITLE
[Bugfix:InstructorUI] fix bugs in dark theme

### PR DESCRIPTION
**Pull request by elihschiff**
### Please check if the PR fulfills these requirements:

* [X] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)

### What is the current behavior?

If an instructor uses custom css the dark mode may have broken some of their css.
The submitty logo may have become very large 
image
as well as if they use a light background, the sidebar buttons might be very hard to see when in dark mode
image

This also fixes an issue where when a user's device defaults them to dark theme (usually on phones with a system wide dark theme) the checkbox to choose the theme is inverted until they click it and refresh the page.

### What is the new behavior?

This fixes the issues listed above.

### CODE REVIEW

**Comment by bmcutler** [comment]
Sort of related question...
Do you think we can get rid of the light vs dark toggle selection in TA grading for code/text files now?  Is there a reason to keep independent control?  (I'd like to make the UI simpler and just have 1 control)

**Comment by bmcutler** [comment]
Also...  the hover color (dark blue) for links is terrible in dark mode.
(e.g. links back to documentation from the instructor's course settings page)

**Comment by elihschiff** [comment]
I removed the button, it now just follows the theme you have set.
The hover link color is now also much easier to see.

File: site/app/templates/misc/SourceSettings.twig [hunk 1]
```diff
@@ -13,7 +13,7 @@
     else {
         editor{{ number }}.setSize("100%", "auto");
     }
-    var theme = localStorage.getItem("codeDisplayStyle");
+    var theme = localStorage.getItem("theme");
```
Comment (MasterOdin) [inline site/app/templates/misc/SourceSettings.twig]: Will this properly account for `window.matchMedia("(prefers-color-scheme: dark)").matches` as well?

**Comment by elihschiff** [comment]
@clino47 and @MasterOdin both your suggestions should now be fixed. Along with a few other minor color issues that I have been informed about.

**Pull request merged**